### PR TITLE
rendering-in-node: add piece about caching

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Adds a section about caching, which is like kinda important for perf / server side rendering :sparkles: - not sure words are on point, re-reads would be heaps valued; hope the message comes across though haha. Thanks!

### CODE REVIEW

File: rendering-in-node.md [hunk 1]
```diff
@@ -74,12 +74,57 @@ else app.start('#app-root'))
...
+- the JS bundles
+- the CSS bundles
+- all the marketing site content
+- every single project README.md page
+
+Stuff you don't want to be caching on the CDN:
+- user specific pages
+- any type of queries
+
+When caching the user specific pages, ideally only the content part would be
```
Comment (laurengarcia) [inline rendering-in-node.md]: wording suggestion: `ideally only the static content would be cached`

File: rendering-in-node.md [hunk 2]
```diff
@@ -74,12 +74,57 @@ else app.start('#app-root'))
...
+cached, leaving out any user-specific data. In order to not confuse people it's
```
Comment (laurengarcia) [inline rendering-in-node.md]: wording suggestions: `leaving out any dynamic content, i.e. user-specific content. For example, in order not to confuse people...`

File: rendering-in-node.md (hunk 1 above)
Comment (yoshuawuyts) [inline rendering-in-node.md]: :100: thanks

File: rendering-in-node.md (hunk 2 above)
Comment (yoshuawuyts) [inline rendering-in-node.md]: solid, thanks!

**Comment by yoshuawuyts** [comment]
Merging!

**Pull request merged**